doc/platforms/gecko.md

```diff
@@ -0,0 +1,104 @@
+Contiki-NG for Silicon Labs (using GSDK)
+=================================
+This guide's aim is to help you with using Contiki-NG for Silicon Labs 
+(using Gecko SDK). 
+
+This port supports the efr32mg12p332f1024gl125 opn and the brd4162a and brd4166a boards.
+
+Port Features
+=============
```

Comment:
```suggestion
## Port Features
```


These sections should be level 2 headings, as this breaks the documentation sidebar:
https://contiki-ng--1845.org.readthedocs.build/en/1845/doc/platforms/gecko.html